README.md

```diff
@@ -1,2 +1,33 @@
-# hw01
-Repository for homework 1
+## Brief Biography 
+
+Hello! My name is Sumer and I am a BA/MA student here at the University of Chicago, studying psychology and computational social science. I am interested in media psychology, especially in how we can harvest digital footprint data to create rich psychological portraits of individuals. I am taking this class to complete the programming requirement for the certificate in computational social sciences. I am originally from **Mumbai, India**, which is a *considerably* warmer place than the frozen tundra that is **Chicago**. I apologize if my formatting seems a *bit* weird but I could not think of a creative way to use **emphasis** in my introduction! If there is anything else you should know about me, it is that I absolutely love books. I love reading non-fiction sampled from popular science, intellectual history and current affairs topics. Here is a list of the books I am currently trying to read, despite the overwhelming coursework: 
+(I have embedded the the Amazon-link into the title of the book)
+
+* [*Black Edge: Inside Information, Dirty Money, and the Quest to Bring Down the Most Wanted Man on Wall Street*](https://amzn.to/2Gsyo0F) by Sheelah Kolhatkar
+
+* [*The Gene: An Intimate History*](https://amzn.to/2EcnslC) by Siddhartha Mukherjee
+
+* [*Englightenment Now: The Case for Reason, Science, Humanism and Progress*](https://amzn.to/2H1Q4kJ) by Steven Pinker
```

Comment:
As well, great touch on including the Amazon links for the books! Definitely, an interesting way to include the required link!